src/DqtApi/Controllers/TeachersController.cs

```diff
@@ -0,0 +1,77 @@
+using System;
+using System.Globalization;
+using System.Text.RegularExpressions;
+using DqtApi.Responses;
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
+using Swashbuckle.AspNetCore.Annotations;
+using DqtApi.DAL;
+using DqtApi.Models;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace DqtApi
+{
+    [ApiController]
+    [Route("v1/teachers")]
+    public class TeachersController : ControllerBase
+    {
+        private readonly IDataverseAdaptor _dataverseAdaptor;
+        public TeachersController(IDataverseAdaptor dataverseAdaptor) {
+            _dataverseAdaptor = dataverseAdaptor;
+        }
+
+        [HttpGet("{trn}")]
+        [ProducesResponseType(typeof(ProblemDetails), StatusCodes.Status400BadRequest)]
+        [ProducesResponseType(typeof(ProblemDetails), StatusCodes.Status404NotFound)]
+        public async Task<IActionResult> GetTeacher(
+            [FromRoute] string trn,
+            [FromQuery(Name = "birthdate"), SwaggerParameter(Required = true)] string birthDate,  // TODO model binder for yyyy-mm-dd format
```

Comment:
I think the built-in model binders will work with this date format
```suggestion
            [FromQuery(Name = "birthdate"), SwaggerParameter(Required = true)] DateTime? birthDate,
```

Reply:
Applied default binding and re-pushed

Reply:
This caused a few tests to fail - issue is that default binding allows other date formats, but they're American, so e.g. birthdate=02-07-1996 will be allowed, but will map to Feb 7th not July 2nd.  If we want to force the yyyy-MM-dd format, I think we need a custom model binder.  Reverting to string for now.